x-pack/plugins/security_solution/cypress/tasks/timeline.ts

```diff
@@ -166,7 +166,8 @@ export const pinFirstEvent = () => {
 
 export const populateTimeline = () => {
   executeTimelineKQL(hostExistsQuery);
-  cy.get(SERVER_SIDE_EVENT_COUNT)
+  cy.get(QUERY_TAB_EVENTS_FOOTER)
+    .find(SERVER_SIDE_EVENT_COUNT)
```

Comment:
if you want to find `SERVER_SIDE_EVENT_COUNT` inside `QUERY_TAB_EVENTS_FOOTER` you should use https://docs.cypress.io/api/commands/within.html#Syntax

Reply:
https://docs.cypress.io/api/commands/find.html#Syntax looks good to me too.
I am using find as the looks flat, however `within` seems always follow by a callback.